back-end/routes/users.js

```diff
@@ -108,16 +117,14 @@ router.post('/', [
       // create user in DB
       dbResults = await createUser(email, hashedPassword);
 
-      let responseBody = await getFullUserProfile(dbResults.id);
-      if (!responseBody) {
-        // no user data could be found
-        return res.status(403).send('User not in database.');
-      }
```

Comment:
Please let me know if I misunderstood, but I felt like this extra call to the database was unnecessary because `createUser` already returns the user data. In this refactor, I use ES6 Object destructuring to grab all of that data from the database call **except** the password and created_at field.